### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by stevekriz**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - run:
+          name: Install Dependencies
+          command: npm ci
+      - save_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
+          paths:
+            - /home/circleci/.npm
+      - run:
+          name: Start Server and Run Tests
+          command: npm start & npm test
```
Comment (BrandonUCSD) [inline .circleci/config.yml:21]: Nicely done! Maybe we can talk about this some time!

File: __tests__/Footer.test.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+let container = null;
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  unmountComponentAtNode(container);
+  container.remove();
```
Comment (BrandonUCSD) [inline __tests__/Footer.test.jsx:15]: what's the purpose of unmounting and removing()? Is this to ensure everything gets removed from container?

File: public/index.html [hunk 3]
```diff
@@ -4,9 +4,8 @@
     <meta charset="utf-8" />
     <link rel="icon" href="Airbnb.ico">
     <title>Vacation Rentals, Homes, Experiences & Places - Airbnb</title>
+    <script src="https://kit.fontawesome.com/bdda782949.js" crossorigin="anonymous"></script>
```
Comment (BrandonUCSD) [inline public/index.html:7]: if you're use SVG's now, do you still need this font awesome?

File: src/app.jsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+const App = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [reviews, setReviews] = useState({});
+  const [displayModal, setDisplayModal] = useState(false);
+  const [viewPortWidth, setViewPortWidth] = useState(1280);
+
+  const getReviews = () => {
+    const random = Math.floor(Math.random() * (100 - 1 + 1) + 1);
+    get(`/api/reviews/${random}`)
```
Comment (BrandonUCSD) [inline src/app.jsx:59]: very cool

**Review by BrandonUCSD — APPROVED** [review]
Overall, looks fine. I'm especially curious about your test methods and would like to talk to you about them some time!

File: .circleci/config.yml (hunk 1 above)
Comment (stevekriz) [reply .circleci/config.yml:21]: Sounds great, anytime!

File: __tests__/Footer.test.jsx (hunk 2 above)
Comment (stevekriz) [reply __tests__/Footer.test.jsx:15]: If you don't perform clean-up and unmount the tree from the document, tests can become "leaky" and one test can change the behavior of another test. That makes them difficult to debug.

File: public/index.html (hunk 3 above)
Comment (stevekriz) [reply public/index.html:7]: Good thinking, however, I still do, for the star.

**Pull request merged**